webapp/src/main/java/com/box/l10n/mojito/Application.java

```diff
@@ -50,6 +52,9 @@ public class Application {
   boolean shouldIndentJacksonOutput;
 
   public static void main(String[] args) throws IOException {
+    // Added a custom model converter to align generated Open API specification schemas with the
+    // actual structure of the requests and responses
+    ModelConverters.getInstance().addConverter(new CustomModelResolver(Json.mapper()));
```

Comment:
This wasn't required in the original reverted commit, can you explain why it's needed?

Reply:
A similar change was required in the original reverted commit. I included additional changes to align the generated Open API specification schemas with the actual request and response structures